gcc/jit/jit-playback.h

```diff
@@ -458,6 +459,18 @@ class compile_to_file : public context
   const char *m_output_path;
 };
 
+class get_target_info : public context
```

Comment:
I think of `get_` prefixes as implying an accessor, rather than something that does a lot of work (which this does).  Please rename this class from `get_target_info` to `populate_target_info` or similar.